### DISCUSSION TITLE
Cable tracing does not work when connecting an interface to a rear port

### DISCUSSION THREAD

**Discussion by romanmendelproject** [netbox-community/netbox#8128] [Q&A]
When connecting the interface to the rear port, the error "path split" appears.
![image](https://user-images.githubusercontent.com/33754158/146922231-d8ab6cd2-990e-4dae-9386-181cd8a5b51d.png)

When you select any node, the cable is not found, although the cable continues to exist.
![image](https://user-images.githubusercontent.com/33754158/146922294-1de18305-4106-48e5-854f-ba337179a60e.png)

This type of connection is needed to connect the interfaces with a hydra cable.

**Comment by DanSheps** [comment]
Connecting a interface to a rear-port with multiple position with multiple corresponding front ports is not supported.
